src/pointer-analysis/value_set.cpp

```diff
@@ -1159,9 +1162,13 @@ void value_sett::assign(
                 "rhs.type():\n"+rhs.type().pretty()+"\n"+
                 "lhs.type():\n"+lhs.type().pretty();
```

Comment:
If we are changing code right next to it, could this become some kind of invariant?

Reply:
Done (`DATA_INVARIANT`).